### PR TITLE
Addw2 support

### PR DESCRIPTION
**Pull request by jackpot51**
Part of https://github.com/system76/firmware-open/pull/84

### CODE REVIEW

File: src/board/system76/common/main.c [hunk 1]
```diff
@@ -74,17 +74,17 @@ void main(void) {
 #endif
 
     // Allow CPU to boot
-    gpio_set(&SB_KBCRST_N, true);
+    //gpio_set(&SB_KBCRST_N, true);
```
Comment (crawfxrd) [inline src/board/system76/common/main.c:77]: Does the EC not have to do all these, or are these disabled for debug/development purposes?

**Comment by jackpot51** [comment]
Tested darp6 and lemp9 successfully - after the last few commits

File: src/board/system76/common/gctrl.c [hunk 2]
```diff
@@ -2,5 +2,8 @@
 
 void gctrl_init(void) {
     SPCTRL1 = 0x03;
+#ifdef POST_DEBUG
+    SPCTRL1 |= 0xC8;
```
Comment (crawfxrd) [inline src/board/system76/common/gctrl.c:6]: This looks specific to 5570.

File: src/board/system76/common/main.c [hunk 3]
```diff
@@ -142,5 +129,15 @@ void main(void) {
         smfi_event();
         // Idle until next timer interrupt
         //Disabled until interrupts used: PCON |= 1;
+
+    #ifdef POST_DEBUG
+        // Read POST codes
+        if (P80H81HS & 1) {
```
Comment (crawfxrd) [inline src/board/system76/common/main.c:135]: This looks specific to 5570.

Comment (jackpot51) [reply src/board/system76/common/main.c:135]: Yeah, that's correct. I can remove it

**Pull request merged**